.aws/src/event-rules/prospect-events/prospectEventRules.ts

```diff
@@ -70,8 +70,13 @@ export class ProspectEvents extends Resource {
 
     // only prod also targets the dev event bridge
     if (!config.isDev) {
-      // TODO: add dev event bridge target
       // https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-cross-account.html
+      // note that permissions have been added by hand to the dev event bus to
+      // allow it to receive events from the prod bus.
+      targets.push({
```

Comment:
do we need a DLQ? 🤔  may be not.. we can just replay the events if they are idempotent on failure.

Reply:
yeah, i don't think we need a DLQ as this isn't a critical flow. also, prospects are generated at least once a day, so if one message fails to reach dev, it's not really going to matter.